### PR TITLE
Fixes #27500 - Use strings to register graphql types

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: app/registries/foreman/plugin/graphql_plugin_fields.rb [hunk 1]
```diff
@@ -6,15 +6,21 @@ module GraphqlPluginFields
...
         end
 
         def realize_plugin_mutation_extensions(source = Foreman::Plugin.graphql_types_registry.plugin_mutation_fields)
           source.map do |plugin_type|
-            send :field, plugin_type[:field_name], mutation: plugin_type[:mutation]
+            send :field, plugin_type[:field_name], mutation: constantize_type(plugin_type[:mutation])
           end
         end
+
+        def constantize_type(string)
```
Comment (timogoebel) [inline app/registries/foreman/plugin/graphql_plugin_fields.rb]: you can use this instead
https://github.com/theforeman/foreman/blob/develop/app/services/foreman/module.rb

**Comment by xprazak2** [comment]
[test foreman]

**Comment by ezr-ondrej** [comment]
Rubocop is complaining:
```
Offenses:

app/models/nic/bmc.rb:24:9: C: Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.
        :password => password_unredacted
```
don't we have an offence in `develop`? :tired_face:

**Comment by ezr-ondrej** [comment]
> don't we have an offence in `develop`?

Nope, it's fixed, you just need to rebase.

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @xprazak2.

**Pull request merged**